x-pack/plugins/actions/server/plugin.ts

```diff
@@ -507,7 +507,6 @@ export class ActionsPlugin implements Plugin<PluginSetupContract, PluginStartCon
       logger,
       eventLogger: this.eventLogger!,
       spaces: plugins.spaces?.spacesService,
-      getActionsClientWithRequest,
```

Comment:
hahaha, I didn't realize we already created an actionsClient here.  In light of https://github.com/elastic/kibana/issues/151620, I wonder if we want to leave this here, as the slack executor will probably need it to re-queue the failed channel requests.  Or not, with the reference to that issue, if we delete here and add back, at least we'll know what to un-do.

Reply:
For sure, let's see what's required for the retries and re-add if we need. If we re-add, we'll have the `alert` SO issue come back and may require a bit more work (like https://github.com/elastic/kibana/pull/151859#discussion_r1115797310).